test/e2e/utils.go

```diff
@@ -0,0 +1,47 @@
+//go:build osde2e
+// +build osde2e
+
+package osde2etests
+
+import (
+	"fmt"
+
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	servicelogsv1 "github.com/openshift-online/ocm-sdk-go/servicelogs/v1"
+	"github.com/openshift/configuration-anomaly-detection/pkg/ocm"
+	ocme2e "github.com/openshift/osde2e-common/pkg/clients/ocm"
+)
+
+// PostStoppedInfraLimitedSupport posts a limited support reason to OCM for a cluster
+// with blocked egress
+func PostStoppedInfraLimitedSupport(clusterID string, ocmCli ocm.Client) error {
```

Comment:
As mentioned above, this is not for the E2E test to do, this is the action we expect from the stage deployed CAD instance.